content/en/dynamic_instrumentation/_index.md

```diff
@@ -66,50 +65,54 @@ To enable Dynamic Instrumentation on a service, select its runtime and follow th
 
 Dynamic Instrumentation can help you understand what your application is doing at runtime. By adding a Dynamic Instrumentation probe you are exporting additional data from your application, without the need to do any code change or redeployment.
 
-### Creating a snapshot probe
-
-A *snapshot probe* exports the context in which it was configured to Datadog. It captures class properties, method arguments and local variables. For more information, read [How Dynamic Instrumentation Works][11].
+### Creating a probe
 
-To create a snapshot probe:
+Both log and metric probes require the same initial setup:
 
 1. Go to the [Dynamic Instrumentation page][12].
-2. Click **Create Probe** in the top right, or click the three dot context menu on a service and select **Add a probe for this service**.
-3. Select **Snapshot** as the probe type.
-4. If not prefilled, choose a service from the list.
-5. If not prefilled, choose runtime, environment and version.
-6. If you set up Source Code Integration for the service, you can select the file where you want to set the probe.
-7. In the source code, select a line on which to set the probe.
+1. Click **Create Probe** in the top right, or click the three dot context menu on a service and select **Add a probe for this service**.
+1. If not prefilled, choose a service from the list.
+1. If not prefilled, choose runtime, environment and version.
+1. In the source code, select a class and method or a source file and line where to set the probe.
+  1. If you set up Source Code Integration for your service, when setting line probes you will receive autocomplete suggestions for the source file and you will see the source code of the selected file when choosing the line.
```

Comment:
I think we better mention here that in order to utilize the Source Code Integration the user has to select specific env and version (rather than "All" option) otherwise we cannot get the matching commit sha, i.e. no code display.

Reply:
We talked about this, we're actually changing the UI to automatically pick the latest version to display source code from - even if the user selects 'all', they will still see something.

I think we show an error in the UI anyway if we don't have enough info to fetch source code, so I think we can leave this as is 🙂